gsutil.py

```diff
@@ -30,6 +30,9 @@
 if (ver.major == 2 and ver.minor < 7) or (ver.major == 3 and ver.minor < 5):
   sys.exit('gsutil requires python 2.7 or 3.5+.')
```

Comment:
Ideally this message would also need to be changed. We should now say something like 3.5-3.11 ?  
We can also consider merging the new if block with this it self ?

Reply:
changed message and merged into one block